Documentation/Books/Manual/SUMMARY.md

```diff
@@ -282,7 +283,7 @@
         * [Sequential Access](Appendix/Deprecated/SimpleQueries/Access.md)
         * [Pagination](Appendix/Deprecated/SimpleQueries/Pagination.md)
         * [Modification Queries](Appendix/Deprecated/SimpleQueries/ModificationQueries.md)
-        * [Geo Queries](Appendix/Deprecated/SimpleQueriesGeoQueries.md)
+        * [Geo Queries](Appendix/Deprecated/SimpleQueries/GeoQueries.md)
```

Comment:
We placed the file directly in the Deprecated folder, because otherwise internal links would be incorrect caused by DocuBlocks. Please make sure all cross links work correctly.

Reply:
As @Simran-B already pointed out: errors like this:

```
not all files of Manual are mapped to the summary!
 files found       |    files in summary
        Appendix/Deprecated/SimpleQueries/Access.md
```

Reply:
That's a different problem if I'm not mistaken. I remember some trouble when DocuBlocks with relative links. They would break depending on the file path depth in the including documents, if their folder level was different - or something along these lines.